### PR TITLE
Fix runtime check of PHP version

### LINKED ISSUE #5465: PHP8 Nullsafe operator gets deleted
## Bug report

* PHP version: 8.0.0
* PHP CS Fixer 2.18.1
* Command used to run: vendor/bin/php-cs-fixer fix --dry-run --diff -vvv
* to fix: Laravel 8 project

### config

```
<?php
{
  -cacheFile: ".php_cs.cache"
  -customFixers: []
  -finder: PhpCsFixer\Finder^ {#847
    -mode: 1
    -names: array:1 [
      0 => "*.php"
    ]
    -notNames: []
    -exclude: array:2 [
      0 => "vendor"
      1 => "/.../someVendorFolder"
    ]
    -filters: []
    -depths: []
    -sizes: []
    -followLinks: false
    -reverseSorting: false
    -sort: false
    -ignore: 3
    -dirs: array:1 [
      0 => ".../FixerTest"
    ]
    -dates: []
    -iterators: []
    -contains: []
    -notContains: []
    -paths: []
    -notPaths: []
    -ignoreUnreadableDirs: false
  }
  -format: "txt"
  -hideProgress: false
  -indent: "    "
  -isRiskyAllowed: false
  -lineEnding: "\n"
  -name: "Laravel"
  -phpExecutable: null
  -rules: array:1 [
    "no_unused_imports" => true
  ]
  -usingCache: true
}
```

### Code Snippet

```php
<?php

namespace App\FixerTest;

use App\Http\Controllers\Controller;
use App\User;
use App\Organization;
use Illuminate\Support\Carbon;

class FixerTest extends Controller
{
    public function nullsafeOperator($id)
    {
        if (User::find($id)->last_active_at?->isAfter(Carbon::now()->subWeek())) {
            return;
        }
    }
}
```

### Expected Outcome

deletion of line `use App\Organization;` since it is a unused import

### Actual Outcome

deletion of unused import and _nullsafe operator_

```
$ vendor/bin/php-cs-fixer fix --dry-run --diff -vvv
PHP CS Fixer 2.18.1 Remote Void by Fabien Potencier and Dariusz Ruminski
Runtime: PHP 8.0.0
Loaded config Laravel from "........./.php_cs".
Using cache file ".php_cs.cache".
F
Legend: ?-unknown, I-invalid file syntax (file ignored), S-skipped (cached or empty file), .-no changes, F-fixed, E-error
   1) app/FixerTest/FixerTest.php (no_unused_imports)
      ---------- begin diff ----------
--- Original
+++ New
@@ @@
 use App\User;
-use App\Organization;
@@ @@
     {
-        if (User::find($id)->last_active_at?->isAfter(Carbon::now()->subWeek())) {
+        if (User::find($id)->last_active_atisAfter(Carbon::now()->subWeek())) {
             return;
         }
     }
 }

      ----------- end diff -----------


Checked all files in 0.025 seconds, 20.000 MB memory used
```

### additional info

`App\User` is an Eloquent model and `last_active_at` is a nullable field.

### PR DESCRIPTION
**Pull request by keradus**
closes #5465

### CODE REVIEW

File: php-cs-fixer [hunk 1]
```diff
@@ -23,7 +23,12 @@ if (defined('HHVM_VERSION_ID')) {
     } else {
         exit(1);
     }
-} elseif (!defined('PHP_VERSION_ID') || \PHP_VERSION_ID < 50600 || \PHP_VERSION_ID >= 80100) {
+} elseif (
+    !defined('PHP_VERSION_ID')
+    || \PHP_VERSION_ID === 80000
```
Comment (hason) [inline php-cs-fixer:28]: This is a bug. Version 8.0.* is supported.

**Review by hason — COMMENTED** [review]
Bug for PHP 8.0.*

File: php-cs-fixer (hunk 1 above)
Comment (kubawerlos) [reply php-cs-fixer:28]: What is a bug? Looks fine.

Comment (keradus) [reply php-cs-fixer:28]: @hason , no.
8.0.0 is not supported due to description from few lines below.
8.0.1+ is supported

**Pull request merged**